internal/services/hpccache/hpc_cache_access_policy_resource.go

```diff
@@ -63,9 +63,9 @@ func resourceHPCCacheAccessPolicy() *pluginsdk.Resource {
 							Type:     pluginsdk.TypeString,
 							Required: true,
 							ValidateFunc: validation.StringInSlice([]string{
-								string(storagecache.Default),
-								string(storagecache.Network),
-								string(storagecache.Host),
+								string(storagecache.NfsAccessRuleScopeDefault),
+								string(storagecache.NfsAccessRuleScopeNetwork),
+								string(storagecache.NfsAccessRuleScopeHost),
```

Comment:
[Old] (https://github.com/Azure/azure-sdk-for-go/blob/f3c695847ff5d4c8ee2b432015a2577dff7b1a53/services/storagecache/mgmt/2021-03-01/storagecache/enums.go#L149-L156)
[New] (https://github.com/Azure/azure-sdk-for-go/blob/f3c695847ff5d4c8ee2b432015a2577dff7b1a53/services/storagecache/mgmt/2021-09-01/storagecache/enums.go#L153-L160)